bats/foreman_helper.bash

```diff
@@ -28,6 +28,12 @@ tKatelloVersion() {
   ) | cut -d. -f1-2
 }
 
+tSkipIfPulp39() {
```

Comment:
```suggestion
tSkipIfPulp339() {
```

Reply:
Thanks Visual Studio Code!  Or, more specifically, the need to *save* files in editors.... :D

Reply:
This _should_ be fixed now. Ctl+s to the rescue!